src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -97,6 +97,13 @@ const ReportActionItemFragment = (props) => {
                 );
             }
 
+            const differByLineBreaksOnly = props.fragment.html.replaceAll('<br />', ' ') === props.fragment.text;
```

Comment:
Can you please add a comment to the code which explains why this logic is necessary?

Reply:
> Can you please add a comment to the code which explains why this logic is necessary?

Updated